colossalai/shardformer/layer/normalization.py

```diff
@@ -34,17 +41,159 @@
     65536,
 ]
 
+class LayerNormBase(nn.Module):
```

Comment:
What is the source of the codes of LayerNormBase, I need to have a check

Reply:
Because both FusedLayerNorm and FusedRMSNorm require overriding the functions named_parameters, named_modules, state_dict, and load_state_dict to correctly handle state dicts, and the code for these functions is repetitive, I placed them in the base class.